ui/src/common/PlayButton.js

```diff
@@ -5,16 +5,8 @@ import { IconButton } from '@material-ui/core'
 import { useDispatch } from 'react-redux'
 import { useDataProvider } from 'react-admin'
 import { playTracks } from '../actions'
-import { makeStyles } from '@material-ui/core/styles'
 
-const useStyles = makeStyles({
-  icon: {
-    color: (props) => props.color,
-  },
-})
-
-export const PlayButton = ({ record, color, size, ...rest }) => {
-  const classes = useStyles({ color })
+export const PlayButton = ({ record, size, className }) => {
```

Comment:
You should remove `color` from the `propTypes` below as well (line 51), and add `className`. Can you also mark `record` as `.isRequired`, please?

Reply:
Making these necessary changes and pushing.